_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
+| {% include anchor a="error.consumption.requests.invalidAcceptParameters" %}                                 | The given AcceptParameters are invalid. ([Request and response introduction]({% link _docs_integrate/request-and-response-introduction.md %}))                                                                                                                                                                                                                                                         |
+| {% include anchor a="error.consumption.requests.invalidRequestItem" %}                                      | The given RequestItem is invalid. Check the error message or the stack trace to learn more.                                                                                                                                                                                                                                                                                                            |
+| {% include anchor a="error.consumption.requests.servalErrorDuringRequestItemProcessing" %}                  | A serialization / validation error occurred during the RequestItem processing. Check the type definitions of your used types.                                                                                                                                                                                                                                                                          |
+| {% include anchor a="error.consumption.requests.unexpectedErrorDuringRequestItemProcessing" %}              | An unexpected error occurred during the RequestItem processing.                                                                                                                                                                                                                                                                                                                                        |
+| {% include anchor a="inheritedFromItem" %}                                                                  | Some child items have errors. If this error occurred during the specification of a Request, call 'validate' to get more information. ([Check if outgoing request can be created]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}), [Test your requests validity]({% link _docs_integrate/integration-example.md %}#connector-test-your-requests-validity)) |
```

Comment:
Change the error message to "error.consumption.validation.inheritedFromItem" if changing base branch from main to release/v5.